src/main/java/ch/uzh/ifi/hase/soprafs23/service/TeamService.java

```diff
@@ -86,4 +88,26 @@ public void changeTurn(int teamId1, int teamId2, int scoredPoints) {
         teamRepository.flush();
         log.debug("Updated points for guessing team and switched roles.");
     }
+
+    public boolean isInTeam(int teamId, String userName) {
```

Comment:
Do we actually get the teamId here? In most cases, we identify the team with a combination of the accessCode and the team number (is also be called team id in a few cases), which is always 1 or two, not matter how many games exist.